### ISSUE TITLE
Error with Crop

### ISSUE THREAD

**Issue by Speepwalker** [darktable-org/darktable#17043]
### Describe the bug

There is something wrong with either the "rotate and perspective" or the "crop" module. If you rotate an image, for example 10 degrees, and later use crop to crop the image, the frame in the crop does not match the actual result.

### Steps to reproduce

Use "Rotate an perspective" to rotate a picture, later crop the image with "crop" module. 

### Expected behavior

Crop to correct size. 

### Where did you obtain darktable from?

downloaded from www.darktable.org

### darktable version

4.8.0

### What OS are you using?

Linux

### What is the version of your OS?

Linux mint 21,3

### Are you using OpenCL GPU in darktable?

I dont know

**Comment by Speepwalker** [comment]
![3](https://github.com/darktable-org/darktable/assets/173609276/e24f3de3-bbc5-4f98-8055-d058ecf0e13c)
![4](https://github.com/darktable-org/darktable/assets/173609276/56a533bd-7a0b-46d7-a7bf-e360d1d61a19)
![2](https://github.com/darktable-org/darktable/assets/173609276/1972b53f-2804-4f23-9e94-f46726b45195)
![1](https://github.com/darktable-org/darktable/assets/173609276/5e7b0ac8-3fb6-47e4-8ab4-623ca98bf6f6)

**Comment by AxelG-DE** [comment]
I have tested this on current master and I can confirm the issue.

**One essential information is missing in above issue report:**
This only happens if crop is set to the aspect ratio "_original image_"

@jenshannoschwalm could this be related to #16976 ?

**Comment by jenshannoschwalm** [comment]
Mmh, i can see your problem but can't reproduce here with images i have. Cou you share raw & xmp?

**Comment by Speepwalker** [comment]
Image and xmp file. peras.net/pp/PAS_1798.CR3, peras.net/pp/PAS_1798.CR3.xmp

**Comment by Speepwalker** [comment]
When the image is in portrait format: The error occurs on the right side of the image.
When the image is in landscape format: The error occurs at the bottom of the image

**Comment by AxelG-DE** [comment]
@Sleepw1 you issue will be solved once PR #17052  is merged into master or 4.8.1

Thank you for reporting!
